### PR TITLE
Update environment.yaml

### PR DESCRIPTION
**Pull request by paul-tqh-nguyen**
Some of the embedding tests use scikit-learn.

In order for us to not skip those tests for the reason that scikit-learn is not installed, we now install it via the environment.yaml.

Those same tests might be skipped because core metagraph doesn't have concrete implementations of certain embedding algorithms.

### CODE REVIEW

**Comment by jim22k** [comment]
The environment.yaml file is not used for the GitHub Actions testing. You need to update the ones inside continuous_integration.

File: metagraph/tests/algorithms/test_embedding.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import networkx as nx
 import numpy as np
 import math
+from sklearn.mixture import GaussianMixture
```
Comment (jim22k) [inline metagraph/tests/algorithms/test_embedding.py:6]: Now that these are required to run the tests, you should update the docs, specifically the installation section.

> Running tests requires additional libraries to be installed

Add scikit-learn to the list of test dependencies needed.

**Review by jim22k — CHANGES_REQUESTED** [review]

File: metagraph/tests/algorithms/test_embedding.py (hunk 1 above)
Comment (paul-tqh-nguyen) [reply metagraph/tests/algorithms/test_embedding.py:6]: https://github.com/metagraph-dev/metagraph/pull/177/commits/55c9a058fda9f51ca569198109d3f15330d1cb50

This commit updates the docs. It notes that we recommend installing `scikit-learn` for testing purposes and updates what an example run of the test suite looks like.

File: environment.yml [hunk 2]
```diff
@@ -25,6 +25,7 @@ dependencies:
   - networkx
   - pandas
   - scipy
+  - scikit-learn
```
[the remark this answers is not in the document]
Comment (paul-tqh-nguyen) [reply environment.yml:28]: https://github.com/metagraph-dev/metagraph/pull/177/commits/73b5a222c2e79b5dbb3852850f6a9e67530d6f95 attempts to update the environment files in `continuous_integration`, but I'm still getting some complaints from GitHub Actions. I'm currently investigating that.

**Comment by jim22k** [comment]
Try adding scikit-learn to continuous_integration/conda/recipe.py under the test/requires section.

File: environment.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (paul-tqh-nguyen) [reply environment.yml:28]: https://github.com/metagraph-dev/metagraph/pull/177/commits/5450caf4e6a4bc828221482abb5cd759237b49b5 adds `scikit-learn` to the `requirements/run` section of the `meta.yaml`. This seems to have fixed things.

**Comment by jim22k** [comment]
@paul-tqh-nguyen, Mind if I take this one from here? I think I figure out how to add a metapackage.

Plus I found some issues (#184) with our recipe that need to be addressed.

**Comment by paul-tqh-nguyen** [comment]
@jim22k , sounds good to me.

**Pull request merged**